src/compute/src/logging/compute.rs

```diff
@@ -54,9 +54,18 @@ pub enum ComputeEvent {
     /// Peek command, true for install and false for retire.
     Peek(Peek, bool),
     /// Available frontier information for dataflow exports.
-    Frontier(GlobalId, Timestamp, i64),
-    // Available frontier information for source instantiations.
-    SourceFrontier(GlobalId, GlobalId, Timestamp, i8),
+    Frontier {
+        id: GlobalId,
+        time: Timestamp,
+        diff: i64,
+    },
+    /// Available frontier information for dataflow imports.
+    ImportFrontier {
+        import_id: GlobalId,
+        export_id: GlobalId,
+        time: Timestamp,
+        diff: i64,
```

Comment:
Note that the change here to `i64` will likely increase the size of the enum, as noted in https://github.com/MaterializeInc/materialize/pull/14031#discussion_r937486429. I think that the impact here is likely to not be very significant, but just wanted to call it out. This was the reason it was an `i8` previously.

Reply:
Ah, good point! I thought that wouldn't matter since an `i8` would just be padded to 8 bytes anyway, but I did not consider the enum tag. I'll make the diffs `i8`s.